.editorconfig

```diff
@@ -0,0 +1,24 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
Just checking this was meant to be committed?

Reply:
Yep, it forces ide formatting